### PR TITLE
Migrated Asset to H5

### PR DESCRIPTION
**Pull request by BenGalewsky**
Picking up from PR #137 
Ran into some Git branch issues and decided the best thing was to abandon the old branch and start over here. Took on Jason's comments from that PR.

### CODE REVIEW

File: psm-app/services/src/main/java/gov/medicaid/entities/Asset.java [hunk 1]
```diff
@@ -15,108 +15,86 @@
...
+public class Asset implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "owner_asset_id")
+    private long id;
 
     /**
      * The asset type.
      */
+    @Column(name ="type")
```
Comment (jasonaowen) [inline psm-app/services/src/main/java/gov/medicaid/entities/Asset.java]: `AssetType` is an `enum`, which I believe means this needs to have the `@Enumerated` annotation to tell Hibernate how to persist it.

Or, now that I look more closely, it doesn't appear that this field is ever used, and the associated `enum` only has a single value, which also isn't ever used. Please just remove both. (In a separate commit - one commit to remove the field & enum, and then the commit to convert the class.)

File: psm-app/services/src/main/java/gov/medicaid/entities/Asset.java [hunk 2]
```diff
@@ -15,108 +15,86 @@
...
      */
+    @ManyToOne
+    @JoinColumn(name = "address_id")
     private Address location;
 
     /**
      * Ownership type.
      */
+    @ManyToOne
+    @JoinColumn(name = "ownership_typ_cd")
```
Comment (jasonaowen) [inline psm-app/services/src/main/java/gov/medicaid/entities/Asset.java]: Please rename this to `ownership_type_code`, and add it to the schema definition in `seed.sql` - I don't see it there now.

File: psm-app/db/seed.sql [hunk 3]
```diff
@@ -1080,3 +1081,12 @@ CREATE TABLE licenses(
...
   attachment_id BIGINT
 );
+
+CREATE TABLE owner_assets(
+  owner_asset_id BIGINT PRIMARY KEY ,
+  type TEXT,
+  name TEXT,
+  address_id BIGINT
+    REFERENCES addresses(address_id),
+  ownership_info_id BIGINT
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: What field does this map to? I don't see any references to it in the class.

File: psm-app/db/seed.sql [hunk 4]
```diff
@@ -1100,3 +1101,11 @@ CREATE TABLE affiliations(
   bgs_study_id TEXT,
   bgs_clearance_date DATE
 );
+
+CREATE TABLE owner_assets(
+  owner_asset_id BIGINT PRIMARY KEY ,
+  name TEXT,
+  address_id BIGINT
+    REFERENCES addresses(address_id),
+  ownership_info_id BIGINT
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: What field does `ownership_info_id` map to? I don't see any references to it in the class.

Comment (BenGalewsky) [inline psm-app/db/seed.sql]: This is a OneToMany reference back to owner info
https://gist.github.com/BenGalewsky/0c33165ac152bd799b872af3000e9b36

**Pull request merged**